src/components/Tabs/components/Item/Item.tsx

```diff
@@ -55,7 +55,9 @@ export default class Item extends React.PureComponent<Props, never> {
     };
 
     const markup = url ? (
-      React.createElement(UnstyledLink, {url, ...sharedProps}, children)
+      <UnstyledLink {...sharedProps} url={url}>
```

Comment:
This didn't fix the error but we should have been doing this, to begin with. IIFC we had a type issue in the past forcing us to use `createElement` that's no longer present.